src/dtx/dtx_rpc.c

```diff
@@ -123,7 +128,8 @@ dtx_drr_cleanup(struct dtx_req_rec *drr)
 		D_FREE(drr->drr_cb_args);
 	}
 	D_FREE(drr->drr_dti);
-	D_FREE(drr->drr_flags);
+	if (drr->drr_bitmap_sz == 0)
+		D_FREE(drr->drr_flags);
```

Comment:
just confirm no need to free drr_bitmap?

Reply:
drr_bitmap is external allocated buffer, will be freed by related (collectively commit/punch DTX) RPC sponsor.